### ISSUE TITLE
Place Focus on the menu list when opening Application Settings Panel

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/thorium-reader#3120]
I am using the keyboard to navigate. 
When I reach the Application Settings Panel, the focus is placed on the close button. One tabulation and I go to "Language". 

Consequently, I have to tab through the entire General Panel to reach the `role="tablist"` element, which acts as a menu. I might never discover that there are more tabs.

The focus should be placed on the `role="tablist"` menu, with the first tab allowing access to the close button.

**Comment by danielweck** [comment]
We need to harmonise React GUI control libraries (Adobe's React ARIA Components vs. Radix which is not quite abandonned but unfortunately not significantly maintained anymore, given that its original creators are implementing an alternative product now)
